### ISSUE TITLE
Unable to launch app w/breakpoints set

### ISSUE THREAD

**Issue by arnjmllr** [microsoft/vscode#232730]
Type: <b>Bug</b>

in the lastest version, when you luanch your app with breakpoints set (debugger mode) the app will never load. I have to unset all of my breakpoints then luanch my app.  once the app is running I can set breakpoints. This does not happen in 1.94.2

VS Code version: Code 1.94.2 (384ff7382de624fb94dbaf6da11977bba1ecd427, 2024-10-09T16:08:44.566Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) Ultra 7 165U (14 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.45GB (17.26GB free)|
|Process Argv|--crash-reporter-id d1806d99-04f0-4806-85a2-91690900924d|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
remotehub|Git|0.64.0
azure-repos|ms-|0.40.0
remote-repositories|ms-|0.42.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
vscrp:30673768
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
bdiig495:31013172
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
wkspc-ranged-t:31151552
cf971741:31144450
iacca2:31156134
notype1cf:31157160
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by Niassamond1** [comment]
i have the same bug, was going insane here. my version is also 1.95.0 was working fine yesterday.

**Comment by Niassamond1** [comment]
i also found something, it works when i have not a single breakpoint, but when i have a breakpoint and the chrome has the sources opened, it opens this weird page : ![Image](https://github.com/user-attachments/assets/6afd7d77-de41-4e6e-9038-affa59730b21)
and debugger is paused cant never play it.

**Comment by Niassamond1** [comment]
i also tried on edge, and get the same error.

**Comment by kbeeveer46** [comment]
Same issue using a React/TypeScript project. When I have the dev tools open when running the project is stops on this. Clicking resume doesn't do anything

![Image](https://github.com/user-attachments/assets/a544147e-8cd0-4d6e-a245-b0f50b38d3a5)

**Comment by Niassamond1** [comment]
Anyone here has any idea how long does it takes for fixes like this? this is making my work a lot harder.

**Comment by dnrr-fd** [comment]
Revert to the last working version and set updates to manual...
https://code.visualstudio.com/updates/v1_93
https://stackoverflow.com/questions/50162685/how-do-i-disable-vs-code-of-updating-itself

**Comment by codicerecta** [comment]
Starting with version 1.95.0, the debugger will hang if you have a breakpoint set. Yes, you can set breakpoints after the app starts but the breakpoint will never be hit. Doesn't matter if you set sourceMaps to true or false. Happens with Chrome and Edge. Debugging is broken in this build. Unless there's a work-around, this release is completely unusable. Too bad because I was starting to enjoy some of the upgrades!

**Comment by wangf1978** [comment]
It is a disaster that this issue happened for 3 days, many issues are reported, but it does not raise any notice, and always mark the issue as "info-need"

**Comment by codicerecta** [comment]
> Revert to the last working version and set updates to manual... https://code.visualstudio.com/updates/v1_93 https://stackoverflow.com/questions/50162685/how-do-i-disable-vs-code-of-updating-itself

Don't forget you'll need to downgrade your extensions separately, such as CoPilot.

**Comment by connor4312** [comment]
https://github.com/microsoft/vscode/issues/232651#issuecomment-2450365808

**Comment by jangopper** [comment]
I do not see how this can be closed. I blamed Chrome - downgraded to 1.04 - the only version that worked somewhat, blamed all; or any of my plugins and uninstalled all and then enabling one by one.

Finally I blamed my Windows installation. After tedious backups I have just installed a new version of Win 11 64, Installed Visual Studio Code from the store - with nothing else installed  - and the issue presents itself with my Laravel Vue SPA and Chrome (and Edge). Firefox showed some life.

Two days of downtime and the issue is closed? #232651 too. Version 1.95 does not work - I had to downgrade to 1.94 which resolved the issue (as mentioned by dnrr-fd above.)
